src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    public double getArea();
```

Comment:
its not neccesary to explicitly declare access modifier, its always public for interface methods
```suggestion
    double getArea();
```